### PR TITLE
fix regression intro'ed w/ auto-fork

### PR DESCRIPTION
**Pull request by maxtaco**
- see commit for breakage: f7b74b9867b4748cc3bd201bee9da8fbffa861a5

### CODE REVIEW

File: go/client/rpc.go [hunk 1]
```diff
@@ -14,25 +14,32 @@ func GetRPCClient() (ret *rpc.Client, xp rpc.Transporter, err error) {
...
 }
 
-func getSocket(g *libkb.GlobalContext) (xp rpc.Transporter, err error) {
+func getSocketWithRetry(g *libkb.GlobalContext) (xp rpc.Transporter, err error) {
+	return getSocket(g, true)
+}
+
+func getSocketNoRetry(g *libkb.GlobalContext) (xp rpc.Transporter, err error) {
+	return getSocket(g, false)
+}
```
Comment (oconnor663) [inline go/client/rpc.go]: I like this style, thanks for hiding the boolean param.

Comment (maxtaco) [inline go/client/rpc.go]: did it for you!

File: go/client/rpc.go [hunk 2]
```diff
@@ -14,25 +14,32 @@ func GetRPCClient() (ret *rpc.Client, xp rpc.Transporter, err error) {
...
-	if xp, err = getSocket(g); err == nil {
+	if xp, err = getSocketNoRetry(g); err == nil {
 		ret = rpc.NewClient(xp, libkb.ErrorUnwrapper{})
 	}
 	return
 }
 
 func GetRPCServer(g *libkb.GlobalContext) (ret *rpc.Server, xp rpc.Transporter, err error) {
-	if xp, err = getSocket(g); err == nil {
+	if xp, err = getSocket(g, false); err == nil {
```
Comment (oconnor663) [inline go/client/rpc.go]: Should this one be `getSocketNoRetry` now?

Comment (maxtaco) [inline go/client/rpc.go]: use the fancy versions from above...

**Comment by oconnor663** [comment]
LGTM with that one change.

**Pull request merged**